R/saveGIF.R

```diff
@@ -83,9 +83,17 @@ saveGIF = function(
   ani.dev = ani.options('ani.dev')
   if (is.character(ani.dev)) ani.dev = get(ani.dev)
   img.fmt = paste(img.name, '%d.', file.ext, sep = '')
-  if ((use.dev <- ani.options('use.dev')))
-    ani.dev(file.path(tempdir(), img.fmt), width = ani.options('ani.width'),
-            height = ani.options('ani.height'))
+
+  if ((use.dev <- ani.options('use.dev'))){
+    if (sum(grepl(ani.options('ani.dev'),c("png","bmp","jpeg","tiff")))){
```

Comment:
I'd use `any()` instead of `sum()`. Although `sum()` works, it feels hackish.

Spaces after commas.